webapp/src/idbworkspace.ts

```diff
@@ -267,12 +288,36 @@ async function getDbAsync(prefix = "__default") {
     }
 }
 
-async function copyTableEntriesAsync(fromDb: pxt.BrowserUtils.IDBWrapper, toDb: pxt.BrowserUtils.IDBWrapper, storeName: string, dontOverwrite: boolean) {
+async function copyTableEntriesAsync(fromDb: pxt.BrowserUtils.IDBWrapper, toDb: pxt.BrowserUtils.IDBWrapper, storeName: string, dontOverwrite: boolean, fromPrefix: string, toPrefix: string) {
+    const migrationDb = await getMigrationDbAsync();
+
     for (const entry of await fromDb.getAllAsync<any>(storeName)) {
+        const key = migrationDbPrefixUpgradeKey(fromPrefix, toPrefix, entry.id);
+        if (await migrationDb.getAsync(storeName, key)) continue;
```

Comment:
Why not have 
```suggestion
        if (await !migrationDb.getAsync(storeName, key)) 
```
and wrap everything underneath it? I feel like I didn't put the `!` in the right place, but just wondering if we can avoid using continue here.

Reply:
this is mostly a stylistic preference, but i think the code is cleaner with the continue

Reply:
As a diehard never-nester, I agree the continue is clearer.